core/src/main/scala/kafka/controller/KafkaController.scala

```diff
@@ -106,6 +97,18 @@ class KafkaController(val config: KafkaConfig,
   private val isrChangeNotificationHandler = new IsrChangeNotificationHandler(eventManager)
   private val logDirEventNotificationHandler = new LogDirEventNotificationHandler(eventManager)
 
+  private val brokerRequestBatch = new ControllerBrokerRequestBatch(config, controllerChannelManager,
```

Comment:
moved all these instantiations below the handlers such that we can have `ReassignmentManager` reference a handler